account_invoice_positive_total/tests/test_account_invoice.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo.tests.common import TransactionCase
+from odoo import exceptions
+
+
+class TestAccountInvoice(TransactionCase):
```

Comment:
Why use TranactionCase and not SavepointCase?
Wouldn't you have faster tests with SavepointCase?